Examples/Modules/RigidInjection/analysis_rigid_injection_LabFrame.py

```diff
@@ -85,5 +85,17 @@ def remove_rigid_lines(plotfile, nlines_if_rigid):
 
 assert( error_rel < tolerance_rel )
 
+
+### Check that user runtime attributes are correctly initialized
```

Comment:
This is a general WarpX comment - something that is missing is documentation about what the CI tests are testing. This is fine that this test is here, but it would be helpful if there was a `README` or some other standardized documentation saying that this rigid injection test is also testing the use of run time attributes. I've noticed this with other tests, that they test multiple things unrelated to the main topic of the test. Though, maybe this comment here is good enough?

Reply:
I admit that I'm guilty of frequently high-jacking some CI random tests to add quick tests of minor features and I have no idea if it's considered good practice. I think that if this part of the test breaks, then it's easy to understand that it's in fact the runtime attributes that are broken (so in that sense I reckon adding this new test is better than nothing).

But it's also true that it's very difficult to know if some minor feature is tested or not in CI.

Regarding the `README`, we often explain the test at the beginning of the analysis script, so maybe that's equivalent. I'll add a sentence at the beginning of the script to state that we also check runtime attributes here.

Reply:
I think that adding extra tests to existing CI tests is ok, and maybe a good thing since it will reduce the overall number of tests. One caveat though is that if a test fails, it should be clear which thing being tested is actually failing.

Having an explanation in the analysis script sounds good to me.